service/internal/auth/casbin.go

```diff
@@ -96,10 +96,12 @@ p,	role:readonly,		/kas.AccessService/LegacyPublicKey,			read,			allow
 ## gRPC routes
 p,	role:unknown,			kas.AccessService/LegacyPublicKey,			other,	allow
 p,	role:unknown,			kas.AccessService/PublicKey,						other,	allow
+p,	role:unknown,			entityresolution.EntityResolutionService.ResolveEntities,										write,		allow
```

Comment:
I could be missing something, but are these public routes, or service to service routes only?

Reply:
the casbin policy yes but auth requires a jwt, right now we dont care about what role the person has, just that a valid jwt is provided

Reply:
its being hit from the opa engine (outside the service)

Reply:
added